README.md

```
@@ -1,8 +1,134 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+<!-- Don't delete it -->
+<div name="readme-top"></div>
+
+<!-- Organization Logo -->
+<div align="center" style="display: flex; align-items: center; justify-content: center; gap: 16px;">
+  <img alt="Stability Nexus" src="public/logo-animated.gif" width="175">
+  <!-- <img src="public/cross.svg" width="30" style="margin: 0;" /> -->
+  <img src="public/hodlcoin.svg" width="175" />
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)